sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1932,7 +1932,8 @@ private Flux<ServiceBusReceivedMessage> nonSessionReactiveReceiveV2() {
 
     Flux<ServiceBusReceivedMessage> nonSessionSyncReceiveV2() {
         assert isOnV2 && !isSessionEnabled;
-        return getOrCreateConsumer().receive();
+        final Flux<ServiceBusReceivedMessage> messages = getOrCreateConsumer().receive();
+        return receiverOptions.isAutoLockRenewEnabled() ? messages.doOnNext(this::beginLockRenewal) : messages;
```

Comment:
In V2 we moved towards dedicated backing methods to support each public receiveMessages variant (sync, async, session-sync, session-async, processor, session-processor) rather than one catch-all backing method like in V1.  So this `nonSessionSyncReceiveV2()` is the backing method for non-session sync receiver.

Unlike V1 there is no need to wrap the messages Flux into FluxAutoLockRenew operator type, a similar lock renew provided by that operator is available in V2 `beginLockRenew` method. The `beginLockRenew` is the one used for V2 Processor and it can be used for lock renewal for V2 sync receive as well.